package.json

```diff
@@ -90,11 +90,13 @@
     "**/graphql-toolkit/lodash": "^4.17.15",
     "**/hoist-non-react-statics": "^3.3.2",
     "**/isomorphic-git/**/base64-js": "^1.2.1",
+    "**/istanbul-instrumenter-loader/schema-utils": "1.0.0",
     "**/image-diff/gm/debug": "^2.6.9",
     "**/load-grunt-config/lodash": "^4.17.20",
     "**/react-dom": "^16.12.0",
     "**/react": "^16.12.0",
     "**/react-test-renderer": "^16.12.0",
+    "**/request": "^2.88.2",
```

Comment:
This selective dependency resolution forces several different versions of request (all between `2.74.0` and `2.88.2`) to resolve to `2.88.2`. According to the [changelog](https://github.com/request/request/blob/master/CHANGELOG.md#v2880-20180810), there are no breaking changes between these versions.